worker/src/unstract/worker/clients/docker.py

```diff
@@ -181,7 +182,12 @@ def get_container_run_config(
                     "target": os.getenv(Env.TOOL_DATA_DIR, "/data"),
                 }
             )
-            envs[Env.EXECUTION_RUN_DATA_FOLDER] = source_path
+            envs[Env.EXECUTION_RUN_DATA_FOLDER] = str(
+                Path(os.getenv(Env.EXECUTION_RUN_DATA_FOLDER_PREFIX, ""))
+                / organization_id
+                / workflow_id
+                / execution_id
+            )
```

Comment:
Use os.path.join instead of using str.
```suggestion
            envs[Env.EXECUTION_RUN_DATA_FOLDER] = os.path.join(
                os.getenv(Env.EXECUTION_RUN_DATA_FOLDER_PREFIX, ""),
                organization_id,
                workflow_id,
                execution_id
            )
```